tests/ui/specs/TimePicker/TimePicker-specs.js

```diff
@@ -0,0 +1,256 @@
+const Page = require('./TimePickerPage');
+const {extractValues, validateTitle} = require('./TimePicker-utils.js');
+
+describe('TimePicker', function () {
+
+	describe('LTR locale', function () {
+		beforeEach(function () {
+			Page.open();
+		});
+
+		describe('default', function () {
+			const timePicker = Page.components.timePickerDefault;
+
+			it('should have correct title', function () {
+				validateTitle(timePicker, 'Time Picker Default');
+			});
+
+			it('should have hour-minute-meridiem order', function () {
+				browser.waitUntil(() => timePicker.decrementer(timePicker.hour).isFocused(), {timeout: 1500,  timeoutMsg: 'hour focused'});
+				Page.spotlightRight();
+				browser.waitUntil(() => timePicker.decrementer(timePicker.minute).isFocused(), {timeout: 1500,  timeoutMsg: 'minute focused'});
+				Page.spotlightRight();
+				browser.waitUntil(() => timePicker.decrementer(timePicker.meridiem).isFocused(), {timeout: 1500,  timeoutMsg: 'meridiem focused'});
+			});
+
+			describe('5-way', function () {
+				it('should decrease the hour when decrementing the picker', function () {
```

Comment:
I got test fail on my local testing. 
Could you please test again on you side?  @daniel-stoian-lgp @adrian-cocoara-lgp

Reply:
I pushed a fix for this bug